### PR TITLE
Ida/test/t217 test add attendees viewmodel

### PR DESCRIPTION
**Pull request by idaoskooei**
coverage:

class 100%
methods 100%
lines 100%

### CODE REVIEW

File: app/src/test/java/com/gathering/android/event/putevent/invitation/AddAttendeesViewModelTest.kt [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+        sut.onViewCreated(attendees, addAttendeeNavigator)
+
+        // WHEN
+        sut.onAttendeeEmailChanged("invalidEmail")
+
+        // THEN
+        val currentState = sut.uiState.value
+        Assert.assertEquals("invalidEmail", currentState.attendeeEmail)
+        Assert.assertFalse(currentState.addAttendeeButtonEnable)
+        Assert.assertEquals("", currentState.errorMessage)
```
Comment (amirziarati) [inline app/src/test/java/com/gathering/android/event/putevent/invitation/AddAttendeesViewModelTest.kt:135]: to confirm is this expected ? no error message with invalid email ?

Comment (idaoskooei) [reply app/src/test/java/com/gathering/android/event/putevent/invitation/AddAttendeesViewModelTest.kt:136]: correct.

File: app/src/test/java/com/gathering/android/event/putevent/invitation/AddAttendeesViewModelTest.kt [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+
+class AddAttendeesViewModelTest {
+
+    @get:Rule
+    val mainDispatcherRule = MainDispatcherRule()
+
+    private lateinit var sut: AddAttendeesViewModel
```
Comment (moyarsuccess) [inline app/src/test/java/com/gathering/android/event/putevent/invitation/AddAttendeesViewModelTest.kt:19]: What is `sut`?

**Review by moyarsuccess — APPROVED** [review]
LGTM :muscle: Only one small suggestion: I noticed you have different naming for test classes with @Animansoubi 's tests. You both have a small talk about that and get sync about the naming.

**Review by amirziarati — APPROVED** [review]

File: app/src/test/java/com/gathering/android/event/putevent/invitation/AddAttendeesViewModelTest.kt (hunk 2 above)
Comment (idaoskooei) [reply app/src/test/java/com/gathering/android/event/putevent/invitation/AddAttendeesViewModelTest.kt:19]: subject under test

**Pull request merged**